this_the_way.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+
+# the function read the path and create list of the files start with deep, then print them
+def thisTheWay():
+    folder_path = input("insert the path to the folder")
+    directory_list = os.listdir(folder_path)
+    only_deep = [f for f in directory_list if f.startswith("deep")]
```

Comment:
Nice work.
The names of some of the variables are not informative enough (f - try not to name variables with only one letter. only_deep - can be more described).